### ISSUE TITLE
Clarification Needed on Subscription Types and Valid Combinations

### ISSUE THREAD

**Issue by sachinvodafone** [camaraproject/Commonalities#84]
According to our documentation, we have outlined two types of subscriptions:

**1. Instance-based subscription :**
   - As per my understanding , this subscription type enables subscribers to receive notifications for events that occur on specific instances of resources.
   - However, our documentation also describes it as "An instance-based subscription is a subscription indirectly created, in addition to another resource creation." This description raises questions about the alignment of these definitions or whether my understanding is wrong. 

**2. Resource-based subscription :**
   - As per my understanding, this subscription allows subscribers to receive notifications for events that occur on all instances of a resource.
   - This definition aligns with our documentation, which states that "A resource-based subscription is an event subscription managed as a resource."

_In addition, we refer to **explicit** and **implicit** subscriptions as methods that allow subscribers to control the way these subscriptions are created. Both can be used in conjunction with **instance-based** and **resource-based** subscriptions and we have made 2 combination as "**Instance-based subscription (implicit creation)**" and "**Resource-based subscription ( explicit creation)**"_ 

To summarise, I would like to verify the accuracy of the definition of instance-based subscription and whether the following two combinations are also valid scenarios:

- Instance-based subscription (explicit creation)
- Resource-based subscription (implicit creation)

**Comment by rartych** [comment]
My understanding is that implicit/explicit is not another dimension, but just other naming for 2 cases we have:

1.   [An instance-based subscription](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#instance-based-implicit-subscription) is a subscription indirectly created, additionally to another resource creation. Another example is in QoD: When QoS session is created (instantiated) API consumer can request to receive notifications related to this session. He **implicitly** subscribes to notifications by providing `webhook` attribute in the POST request  creating this session.
2. [A resource-based subscription](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#resource-based-explicit-subscription) is an event subscription managed as a resource. Here we can look at Device Location: in geo-fencing use case there is no resource created for device location, instead API consumer creates subscription for events related to device leaving or entering the specified zone. The `subscriptions/{id}` resource is created, so the subscription is **explicit**.

Probably we need to precise it in the Design Guidelines, maybe there are better names describing these two cases without confusion.
You can add proposed clarifications in the issue https://github.com/camaraproject/Commonalities/issues/83 and the related to it PR (coming soon).

**Comment by sachinvodafone** [comment]
I am clear on this and can close this issue.
